src/Componenets/Header/Header.css

```diff
@@ -46,3 +46,13 @@ img {
 .navbar-brand {
   width: 55%;
 }
+
+@media only screen and (max-width: 768px) {
```

Comment:
Hey @muditasisodia, alignment of the nav links is right now. But the logo is becoming a little bit small of resizing. I was wondering if you can fix that too. :innocent: 
![screenshot from 2018-04-16 11-04-31](https://user-images.githubusercontent.com/10762218/38791575-36e01e70-4166-11e8-8cce-36a6243b1743.png)